app/controllers/admin/sales/invoices.js

```diff
@@ -39,7 +38,11 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
         name      : 'Status',
         valuePath : 'status'
       }
+      //{
+      //  name            : 'Action',
+      //  valuePath       : 'identifier',
+      //  cellComponent   : 'ui-table/cell/admin/sales/cell-action'
+      //}   
```

Comment:
Codacy found an issue: [Trailing spaces not allowed.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6173716)